assets/css/structure.css

```diff
@@ -0,0 +1,32 @@
+.section {
+  border: var(--border);
+  margin: var(--large);
+  padding: var(--large);
+  background-color: var(--background-color);
+}
+
+.quote {
+  font-weight: var(--normal-font);
+  margin-bottom: var(--large);
+  color: var(--primary-color);
+}
+
+.author {
+  font-weight: var(--bold-font);
+  text-align: right;
+  margin-bottom: var(--large);
+  color: var(--primary-color);
+}
+
+.list {
+  padding-top: var(--large);
+}
+
+.list-item {
+  display: inline;
+  font-weight: var(--thin-font);
+  margin: var(--small);
+  padding: var(--xsmall);
+  color: var(--secondery-color);
+  border: var(--border);
+}
```

Comment:
I suggest using a better name like: `tag-list` and `tag`. `list` is a very generic name.

Reply:
Done.